account_financial_report_qweb/README.rst

```diff
@@ -14,9 +14,16 @@ Accounting / Reporting / OCA Reports.
 - Open Items
 - Aged Partner Balance
 
+Usage
+=====
+
+You can filter accounts by tags or with a One2many field in General Ledger and
```

Comment:
Then the tags should be a included in the search view, so that the user does not have to use the "Add custom filter" option.

Reply:
@jbeficent sorry by the confusion, I only want change the field widget from many2many_tags to One2many_tags, but I did not know about *web_widget_many2many_tags_multi_selection* module, I think that the migration of this module is the better solution, so when it is migrated I will close the PR.
Thanks.